CiBuild.cmd

```diff
@@ -1,31 +0,0 @@
-@if not defined _echo echo off
```

Comment:
This should not be deleted, it's used on the `netci.groovy` (which might also need to be updated).